src/giphy-gifme.coffee

```diff
@@ -9,25 +9,95 @@
 # process.env.HUBOT_GIPHY_RATING = 'pg' (y,g, pg, pg-13 or r)
 #
 # Commands:
-# hubot gif me (tag 1, tag 2) - Get a random gif (tagged with "tag 1" and "tag 2")
+# hubot gif me - Get a completely random GIF
+# hubot gif me tag 1, tag 2 - Search for a GIF tagged with "tag 1" and "tag 2"
+# hubot gif me "tag 1" "tag 2" - Search for a GIF tagged with "tag 1" and "tag 2"
+# hubot gif me tag1 tag2 - Search for a GIF tagged with "tag1" and "tag2"
 #
 # Author:
 # Ben Centra
 
-api_key = process.env.HUBOT_GIPHY_API_KEY or 'dc6zaTOxFJmzC' # <== Giphy's public API key, please request your own!
+# Giphy's public API key; please request your own!
+api_key = process.env.HUBOT_GIPHY_API_KEY or 'dc6zaTOxFJmzC'
+
+# Content rating limit:
```

Comment:
Forgot to add valid content ratings to the comment on line 23... just a little too hasty with this PR, I think. If you'd like to correct it, valid values are: y,g, pg, pg-13 or r